File: src/main/java/me/kuku/yuq/service/MessageService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.MessageEntity;

import java.util.List;
import java.util.Map;

@AutoBind
public interface MessageService {
    MessageEntity findByMessageId(int messageId);
    void save(MessageEntity messageEntity);
    Map<Long, Long> findCountQQByGroupAndToday(Long group);
    List<MessageEntity> findLastMessage(Long qq, Long group);
    List<MessageEntity> findByGroupExcludeQQ(Long group, Long qq);
}

File: src/main/kotlin/me/kuku/yuq/service/MessageService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.MessageEntity

@AutoBind
interface MessageService {
    fun findByMessageId(messageId: Int): MessageEntity?
    fun save(messageEntity: MessageEntity)
    fun findCountQQByGroupAndToday(group: Long): Map<Long, Long>
}